rest_framework/views.py

```diff
@@ -425,17 +425,18 @@ def handle_exception(self, exc):
             else:
                 exc.status_code = status.HTTP_403_FORBIDDEN
 
-        exception_handler = self.settings.EXCEPTION_HANDLER
-
         context = self.get_exception_handler_context()
-        response = exception_handler(exc, context)
+        response = self._get_error_response(exc, context)
 
         if response is None:
             raise
 
         response.exception = True
         return response
 
+    def _get_error_response(self, exc, context):
```

Comment:
it returns a response in this case, not the function itself, so I picked different name. As for private - depends how you want it. For me it's not something you need as public as it's for specific internal use.